mvc/config/config.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+	//databse configuration 
+	define("HOST","localhost");
+	define("USERNAME","root");
+	define("PASSWORD","Rajdip@2001");
+	define("DATABASE","indibook");
+	define("BASEURL", "http://indibook.com");
```

Comment:
This is not a part of the database configuration so we could separate the line using a blank line.